packages/icons/scripts/build.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs');
+const glob = require('glob');
+const SVGO = require('svgo');
+const path = require('path');
+const prettier = require('prettier');
+const camelCase = require('lodash.camelcase');
+
+const svgo = new SVGO({});
+
+const svgPaths = glob.sync('./svg/*');
+const outputDir = './react/';
+
+const prettierOptions = { parser: 'babel', singleQuote: true };
```

Comment:
Maybe add all options we use in [the rest of our codebase](https://github.com/codesandbox/codesandbox-client/blob/62e179b292e629e8a4cf28205b8b6c6136e0636c/.prettierrc)? 🤔

```suggestion
const prettierOptions = {
  endOfLine: 'lf',
  parser: 'babel',
  singleQuote: true,
  trailingComma: 'es5',
};
```

Or maybe get it from the root's `.prettierrc`? 🤔